sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -1513,9 +1517,15 @@ pi_result piextProgramSetSpecializationConstant(pi_program, pi_uint32, size_t,
   DIE_NO_IMPLEMENTATION;
 }
 
-pi_result piextDeviceSelectBinary(pi_device, pi_device_binary *, pi_uint32,
-                                  pi_uint32 *) {
-  DIE_NO_IMPLEMENTATION;
+pi_result piextDeviceSelectBinary(pi_device, pi_device_binary *,
+                                  pi_uint32 RawImgSize, pi_uint32 *ImgInd) {
+  /// TODO : Support multiple images and enable selection algorithm
+  /// for the images
+  if (RawImgSize != 1) {
+    return PI_INVALID_VALUE;
```

Comment:
It is better to to output a message instead of silent returning PI_INVALID_VALUE so that user has a clue what went wrong.

Reply:
Done.